src/reports.cpp

```diff
@@ -353,6 +353,14 @@ static config unit_status(reports::context & rc, const unit* u)
 		add_status(res, "misc/petrified.png", N_("petrified: "),
 			N_("This unit has been petrified. It may not move or attack."));
 	}
+	if (u->get_state(unit::STATE_UNHEALABLE)) {
+		add_status(res, "misc/unhealable.png", N_("unhealable: "),
+			N_("This unit is unhealable. It cannot be healed by healers or villages and doesn't benefit from resting."));
```

Comment:
`doesn't` needs to use the typographic apostrophe: @Pentarctagon is there a string freeze current?

Reply:
Nope.

Reply:
Thanks. 90ef6bcc23606ac03b411a90436dec505a6a2f13